pages/dao/[symbol]/proposal/components/instructions/Raydium/AddLiquidityToPool.tsx

```diff
@@ -0,0 +1,159 @@
+      liquidityPool: yup.string().required('Liquidity Pool is required'),
+      baseAmountIn: yup
+        .number()
+        .moreThan(0, 'Amount for Base token should be more than 0')
+        .required('Amount for Base token is required'),
+      quoteAmountIn: yup
+        .number()
+        .moreThan(0, 'Amount for Quote token should be more than 0')
+        .required('Amount for Quote token is required'),
+      fixedSide: yup
+        .string()
+        .equals(['base', 'quote'])
+        .required('Fixed Side is required'),
+      slippage: yup.number().required('Slippage value is required'),
+    }),
+    buildInstruction: async function () {
+      if (!form.liquidityPool || !form.baseAmountIn || !form.quoteAmountIn)
+        throw new Error('missing field in instruction form')
+      const poolKeys = getLiquidityPoolKeysByLabel(form.liquidityPool)
+      const [base, quote] = await Promise.all([
+        connection.current.getTokenSupply(poolKeys.baseMint),
+        connection.current.getTokenSupply(poolKeys.quoteMint),
+      ])
+      return createAddLiquidityInstruction(
+        poolKeys,
+        uiAmountToNativeBN(form.baseAmountIn, base.value.decimals),
+        uiAmountToNativeBN(form.quoteAmountIn, quote.value.decimals),
+        form.fixedSide,
+        form.governedAccount!.governance.pubkey
+      )
+    },
+  })
+
+  useEffect(() => {
+    debounce.debounceFcn(async () => {
+      if (!form.baseAmountIn || !form.liquidityPool) return
+
+      handleSetForm({
+        value: await getAmountOut(
+          form.liquidityPool,
+          form.baseAmountIn,
+          connection.current,
+          form.slippage
+        ),
+        propertyName: 'quoteAmountIn',
+      })
+    })
+  }, [form.baseAmountIn, form.slippage])
+
+  return (
+    <>
+      <Select
+        label="Raydium Liquidity Pool"
+        value={form.liquidityPool}
+        placeholder="Please select..."
+        onChange={(value) =>
+          handleSetForm({ value, propertyName: 'liquidityPool' })
+        }
+        error={formErrors['liquidityPool']}
+      >
+        <SelectOptionList list={Object.keys(liquidityPoolKeysList)} />
+      </Select>
+
+      {form.liquidityPool && (
+        <>
+          <Input
+            label="Base Token Amount to deposit"
+            value={form.baseAmountIn}
+            type="number"
+            min={0}
+            onChange={(evt) =>
+              handleSetForm({
+                value: evt.target.value,
+                propertyName: 'baseAmountIn',
+              })
+            }
+            error={formErrors['baseAmountIn']}
+          />
+
+          <Select
+            label="Slippage (%)"
+            value={form.slippage}
+            onChange={(value) =>
+              handleSetForm({ value, propertyName: 'slippage' })
+            }
+            error={formErrors['slippage']}
+          >
+            <SelectOptionList list={[0.5, 1, 2]} />
```

Comment:
put available values in their own variable at top of file like `SLIPPAGE_OPTIONS = [...]`